### ISSUE TITLE
No such file or directory 

### ISSUE THREAD

**Issue by thegeorgeous** [therockmandolinist/emacs-hacker-typer#3]
Running `M-x hacker-typer` returns this error

```
mm-write-region: Opening output file: No such file or directory, /Users/thegeorgeous/.emacs.d/hacker-typer/xlwmenu.c
```

**Comment by dieggsy** [comment]
Thanks for the issue! I actually realized this as soon as the package was accepted, I'm failing to create the `hacker-typer/` directory if `no-littering` is not present. I was working on some improvements to the package where this should be fixed, should be out within the next day or so.

**Comment by dieggsy** [comment]
Should be fixed by 2f2ec41460ea88bc77c60539ea7646959f2da24b, let me know.

**Comment by thegeorgeous** [comment]
Works now! 👍
